javaOOP_polymorphism/src/jv/oop/lab/math/operation/MathOperation.java

```diff
@@ -0,0 +1,15 @@
+package jv.oop.lab.math.operation;
+
+public class MathOperation {
+	public int add(int a, int b) {
+		return a + b;
+	}
+
+	public int add(int a, int b, int c) {
+		return add(a, b) + c;
+	}
+
+	public int add(int a, int b, int c, int d) {
+		return add(a, b, c) + d;
+	}
```

Comment:
I see this is lab, but it could be only one method:
	public int add(int... numbers) {
		int sum = 0;
		for (int number : numbers) {
			sum += number;
		}
		return sum;
	}

Reply:
it was part of the explanation of the polymorphism